src/mscorlib/src/System/TimeZoneInfo.cs

```diff
@@ -1420,7 +1422,7 @@ private class OffsetAndRule {
             return localConverted;           
         }
 
-
+#if FEATURE_WIN32_REGISTRY
```

Comment:
It's not :smile: https://github.com/dotnet/coreclr/blob/9d1baf5921040dd364fc7cbf9271541c4a26c287/clr.coreclr.props#L108